install/local_mac_os/05_open_browser.md

```diff
@@ -9,6 +9,8 @@
 
 <!-- /TOC -->
 
+<!-- /TOC -->
+
```

Comment:
roll back

Reply:
Handled in [0ae93f4](https://github.com/ajktown/docs/pull/65/commits/0ae93f4a216a68affc30883ea5728bbc8364bcd2)